<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diaryManagement</class>
 <widget class="QWidget" name="diaryManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>472</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListView" name="diaryView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>501</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="diarySearch">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboMethod">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>按地点搜索</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按标题搜索</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按内容搜索</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboSort">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>无</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>评分前十</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>人气前十</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>按兴趣的人气前十</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnSearch">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>搜索方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>排序方式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>兴趣偏好</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboInterest">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>科技</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>艺术</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>教育</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>医学</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>金融</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>划船</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>登山</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>徒步</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="creatBnt">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>创建日记</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBnt">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
